plugins/citadel-paper/src/main/java/vg/civcraft/mc/citadel/CitadelUtility.java

```diff
@@ -212,6 +215,26 @@ public static boolean attemptReinforcementCreation(Block block, ReinforcementTyp
             Citadel.getInstance().getLogger().info(player.getName() + " created reinforcement with " + type.getName()
                 + " for " + block.getType().toString() + " at " + block.getLocation().toString());
         }
+        // warn if acid cannot complete
+        if (Citadel.getInstance().getAcidManager().isPossibleAcidBlock(block)) {
+            AcidManager acidMan = Citadel.getInstance().getAcidManager();
+            AcidType acidType = acidMan.getAcidTypeFromMaterial(block.getType());
+            for (BlockFace blockFace : acidType.blockFaces()) {
+                Block relativeBlock = block.getRelative(blockFace);
+                Reinforcement relativeReinforcement = ReinforcementLogic.getReinforcementProtecting(relativeBlock);
+                if (
+                    relativeReinforcement == null
+                        || !relativeReinforcement.getType().canBeReinforced(relativeBlock.getType())
+                        || acidMan.isPossibleAcidBlock(relativeBlock)
+                ) {
+                    continue;
+                }
+                if (!acidMan.canAcidBlock(type, relativeReinforcement.getType())) {
+                    CitadelUtility.sendAndLog(player, ChatColor.RED,
```

Comment:
this message should only be sent once (by exiting the loop with break)

Reply:
its a bit intentional since there could be situations where multiple acid operations could fail but one could still be valid

Reply:
hm ok this is probably fine